homeassistant/components/honeywell/__init__.py

```diff
@@ -106,6 +106,7 @@ def _refresh_devices(self):
         """Refresh each enabled device."""
         for device in self.devices:
             device.refresh()
+            time.sleep(3)
```

Comment:
Ooooh. Aaaah. I know, but this is the only way I can think of to not send X number of requests off to the API all at once. Most people have < 10 thermostats and this in run in an executor job. Hopefully this is okay as a temporary fix?

Reply:
No, we don't allow sleeping in the executor thread pool. If we need to sleep we need to do it with asyncio in async context.

Reply:
Fixed, switched it over to using `asyncio.sleep` instead.